src/data-table/DataTable.tsx

```diff
@@ -117,6 +118,7 @@ export function DataTable<T extends ReferenceObject = TableObject>(props: DataTa
         sorting: controlledSorting,
         selection: controlledSelection,
         pagination: controlledPagination,
+        pageSizeOptions = [10, 25, 50, 100],
```

Comment:
Typically, we'd like to set also the default value. Maybe we can expand that prop: `type paginationOptions = {pageSizeOptions: [10, 20], pageSizeInitialValue: 20}`, or something like that?